calico_node/Dockerfile

```diff
@@ -33,10 +38,17 @@ RUN apk add --no-cache ip6tables ipset iputils iproute2 conntrack-tools
 # Install Python so libnetwork-plugin can run
 RUN apk --no-cache add python py-setuptools
 
-# Actually install libnetwork-plugin. A python dev environment (and git) is installed so libnetwork-plugin can be pip installed.
+# Copy "shared directory" which can be used to pass custom files (local git repos,
+# that can be used by pip install git+file:///)
+COPY node_share /tmp/node_share
```

Comment:
This is the only part I'm not sure about.  We use this docker image for releases too so I'm wary of including this directory in a release by accident.  @tomdee can you suggest a better way of making this work for pip installing from a file URL?